docs/topic_guides/evaluation/model.md

```diff
@@ -1,3 +1,33 @@
 # Model Evaluation
 
-This page is under construction - check back soon!
\ No newline at end of file
+The parameters of a trained Splink model build up a set of rules for scoring any pairwise records comparisons. Before scoring any pairs of records there are a number of ways to check whether your model will perform as you expect.
+
+## Look at the model parameters
+
+The final model is summarised in the [match weights chart](../../charts/match_weights_chart.ipynb) with each bar in the chart signifying the match weight (i.e. the amount of positive or negative evidence for a match) for each comparison level in your model.
```

Comment:
This might read slightly better?

```suggestion
The final model is summarised in the [match weights chart](../../charts/match_weights_chart.ipynb) with each bar in the chart signifying the match weight (i.e. the amount of evidence for or against a match) for each comparison level in your model.
```